bot/cogs/snakes.py

```diff
@@ -40,10 +153,28 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        if name is None:
+            name = random.choice(snakelist)
+        elif name == "snakes on a plane":
+            await ctx.send("https://media.giphy.com/media/5xtDartXnQbcW5CfM64/giphy.gif")
+        elif name == "python":
+            with open(SNEKFILE, 'r') as file:
+                text = file.read()
+                snake_embed = Embed(color=ctx.me.color, title="SNEK")
+                snake_embed.add_field(name="Python", value=f"*{text}*")
+                snake_embed.set_thumbnail(url="http://www.pngall.com/wp-content/uploads/2016/05/Python-Logo-Free-PNG-Image.png")
+                await ctx.send(embed=snake_embed)
+
+        snake = await self.get_snek(name)
+        snake_embed = Embed(color=ctx.me.color, title="SNEK")
+        snake_embed.add_field(name=snake['name'], value=snake['snake_text'])
+        snake_embed.set_thumbnail(url=snake['snake_image'])
+        await ctx.send(embed=snake_embed)
 
     # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
 
 
 def setup(bot):
     bot.add_cog(Snakes(bot))
     log.info("Cog loaded: Snakes")
+    bot.loop.create_task(Snakes(bot).cache_snakelist())
```

Comment:
Instead of creating two instances of the same class, the `__init__` method should create the task to cache the data.

Reply:
Runew0lf wrote it.

Reply:
Ruinew0lf fixed it.